self-core-impl/src/test/java/com/selfxdsd/core/contributors/ProjectContributorsTestCase.java

```diff
@@ -494,6 +517,11 @@ private Project mockProject(
         Mockito.when(wallet.available()).thenReturn(budget);
         Mockito.when(project.wallet()).thenReturn(wallet);
 
+        final ProjectManager projectManager = Mockito
+            .mock(ProjectManager.class);
+        Mockito.when(projectManager.commission()).thenReturn(pmCommission);
+        Mockito.when(project.projectManager()).thenReturn(projectManager);
+
         return project;
     }
 }
```

Comment:
@criske can you add a unit test where all the contributors have an affordable price, but it goes our of range because of the commission? 

For example, Wallet has 15000 available cents, the Contributor's price is 14500, and the commission is 600. Then the total price (15100) is our of rance :D